Added two configurations for commercial IAST tools.

rewtd opened the pull request:
This adds the config required for the commercial IAST tools Seeker and CxIAST (without adding their actual code to the repository).

cx-jason commented:
hi Grant

may i understand from you, why are you editing the source file "CheckmarxIASTReader.java" ?
thanks

rewtd commented:
Hi Jason,

I didn't actually. I added that file myself (it didn't exist in the repo
when I initially pulled it) but it seems that someone else also added it?

Best regards,
Grant



On Mon, 9 Mar 2020 at 02:16, Jason Khoo <notifications@github.com> wrote:

> hi Grant
>
> may i understand from you, why are you editing the source file
> "CheckmarxIASTReader.java" ?
> thanks
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/OWASP/Benchmark/pull/99?email_source=notifications&email_token=ABORWTYMQ3UTXPQS4HL6BE3RGRGOZA5CNFSM4LDUDCR2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEOFNMWI#issuecomment-596301401>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABORWT3Z7JBN3GUPS5N6CNDRGRGOZANCNFSM4LDUDCRQ>
> .
>

davewichers commented:
Grant,  these look like excellent contributions. Thank you.

However you and Jason both contributed a CheckmarxIASTReader.  Can you resolve the conflict with that file, and BenchmarkScore, so I can just click/accept your additions/changes?

cx-jason commented:
hi Grant
appreciate your contribution.
please do not modified the source file "CheckmarxIASTReader.java" because this my current contribution project
thanks

rewtd commented:
Conflicts resolved in consultation with @cx-jason
